src/app/courses/courses-header/courses-header.component.spec.ts

```diff
@@ -22,4 +26,20 @@ describe('CoursesHeaderComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it ('should search when clicked', () => {
+    const spy = spyOn(component, 'searchCourses');
+    fixture.detectChanges();
+
+    fixture.debugElement.query(By.css('.search-course .btn')).triggerEventHandler('click', null);
+
+    expect(spy).toHaveBeenCalled();
+  });
+
+  it ('should log message', () => {
```

Comment:
probably good for education, but I'd really won't do that, prev test is enough here